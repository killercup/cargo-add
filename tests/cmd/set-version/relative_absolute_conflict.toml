bin.name = "cargo-set-version"
args = ["set-version", "1.0.0", "--bump", "major"]
status.code = 2
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
