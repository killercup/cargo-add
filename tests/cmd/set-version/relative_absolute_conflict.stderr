error: The argument '<TARGET>' cannot be used with '--bump <BUMP>'

USAGE:
    cargo set-version <TARGET>

For more information try --help
